docs/sphinx/source/tuf.ngclient.rst

```diff
@@ -0,0 +1,10 @@
+Client API
+==========
```

Comment:
This page seems to have very little value as is... I wonder if we should add a short explanation here already or just remove this and direct people to updater directly?

Reply:
Also, if you name this "Client API" then the links will be called that as well -- so the modern-implementation page will now talk about a "Client API module"...